android/PhysicalWeb/app/src/main/res/layout/list_item_nearby_beacon.xml

```diff
@@ -21,6 +21,22 @@
                 android:layout_width="0dp"
                 android:layout_height="wrap_content"
                 android:orientation="vertical"
+                android:layout_weight="1"
+                android:gravity="end"
+                android:layout_marginRight="23dp">
```

Comment:
This shouldn't be needed.  But in case it is: use layout_marginEnd instead of layout_marginRight.  In general you should use the Start/End versions of these attributes to support RTL languages.